psutil/__init__.py

```diff
@@ -1522,8 +1525,8 @@ def cpu_times_percent(interval=None, percpu=False):
     def calculate(t1, t2):
         nums = []
         all_delta = sum(t2) - sum(t1)
-        for field in t1._fields:
-            field_delta = getattr(t2, field) - getattr(t1, field)
+        for field in t2._fields:
```

Comment:
`t1` is the least-recent set of data points, so if the underlying /proc/stat file changes to provide a different number of fields (e.g. if we go from not having fields to having some fields) then `t2` will have the latest and most correct set of fields.

The danger is that `t1` will be missing some values, hence the default attr provided on the next line.